README.md

```diff
@@ -2,20 +2,30 @@
 
 ## Introduction
 
-The `navigator.printing` Web API is a proposed standard for enabling printer-related functionality in web applications. Built on top of the Internet Printing Protocol (IPP), this API provides a set of JavaScript methods that allow developers to query local printers, submit print jobs, and manage print job options and status directly from web applications.
+The `navigator.printing` Web API is a proposed standard for enabling printer-related functionality in web applications. This API provides a set of JavaScript methods that allow developers to query local printers, submit print jobs, and manage print job options and status directly from web applications. To represent these concepts, it relies on the attribute names and semantics from the IPP printing standards.
 
 Printing is a common and important task in many domains, including document management, label printing, receipt printing, and more. While web applications can generate printable content such as PDFs, images, and text documents, sending print jobs from web applications to local printers has traditionally required various workarounds such as relying on third-party plugins or polyfilling via `window.print()`; `navigator.printing` API aims to provide a standardized and seamless way for web applications to directly interact with printers, enabling developers to build printer-related features within their web applications.
 
 The explainer outlines the motivating use cases for the API, its key features, and how it can benefit web developers and users alike.
 
 ## Motivating Use Cases
 
-### Augmenting `window.print()`
+Although `window.print()` offers basic printing functionality through the browser's default print dialog, it is a relatively simple API. Developers' capabilities are restricted to selecting a file for printing and triggering the print dialog, which places the decision-making responsibility solely on the end user. To address these limitations and facilitate advanced use cases, we aim to provide developers with greater control over the printing process and expand the range of options available to them.
```

Comment:
Giving control to the end user is generally seen as a good thing. Can you rephrase this to explain how users benefit from giving some of this control to the application developer instead?

Reply:
Added a few examples.